### PR TITLE
feat(checkbox): add `kv-checkbox` component

### PR DESCRIPTION
**Pull request by djcouto**
Screenshot 2023-01-20 at 14 15 04

### CODE REVIEW

**Comment by GoncaloCamaz** [comment]
Can you please add a print of the new component? 😃

**Review by GoncaloCamaz — APPROVED** [review]

File: packages/react-ui-components/src/stories/Components/Checkbox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	title: 'Inputs/Checkbox',
+	component: 'kv-checkbox',
+	parameters: {
+		notes: require('@ui-notes/checkbox/readme.md')
+	}
+};
+
+KvCheckbox.displayName = 'Checkbox';
+
+const CheckboxTemplate: ComponentStory<typeof KvCheckbox> = args => {
```
Comment (laraujo7) [inline packages/react-ui-components/src/stories/Components/Checkbox.stories.tsx:15]: Couldn't we do `const CheckboxTemplate: ComponentStory<typeof KvCheckbox> = ({ checked: initialChecked, ...otherArgs }) =>{`  ?

**Review by laraujo7 — APPROVED** [review]

**Review by ricardogoncalves89 — APPROVED** [review]
👌

File: packages/react-ui-components/src/stories/Components/Checkbox.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export default {
+	title: 'Inputs/Checkbox',
+	component: 'kv-checkbox',
+	parameters: {
+		notes: require('@ui-notes/checkbox/readme.md')
+	}
+};
+
+KvCheckbox.displayName = 'Checkbox';
```
Comment (ricardogoncalves89) [inline packages/react-ui-components/src/stories/Components/Checkbox.stories.tsx:13]:
```suggestion
KvCheckbox.displayName = 'KvCheckbox';
```

**Pull request merged**